src/modules/person/index.ts

```diff
@@ -100,18 +93,11 @@ export class PersonModule extends ModuleBase {
    * @since 8.0.0
    */
   firstName(sex?: SexType): string {
-    const { first_name, female_first_name, male_first_name } =
-      this.faker.rawDefinitions.person ?? {};
-
     return selectDefinition(
       this.faker,
       this.faker.helpers.arrayElement,
       sex,
-      {
-        generic: first_name,
-        female: female_first_name,
-        male: male_first_name,
-      },
+      this.faker.rawDefinitions.person?.first_name,
```

Comment:
This should now use the main definitions, since we no longer have to check whether any gendered variant is present.

```suggestion
      this.faker.definitions.person.first_name,
```